remotes/locationIndependentIdentifier.json

```diff
@@ -0,0 +1,11 @@
+{
+    "refToInteger": {
+        "$ref": "#foo"
+    },
+    "definitions": {
+        "A": {
+            "id": "#foo",
+            "type": "integer"
+        }
+    }
```

Comment:
it doesn't make much sense to me that this file has a schema on a properties of the root (like subSchemas.json) but the root also seems to be a schema, given the structure of `definitions`. it's probably better to put `refToInteger` in `definitions` too.

Reply:
I changed that in 9c4d8b10cef7aece339115e0d65e2d4b6a0a9b69